pyaedt/application/Variables.py

```diff
@@ -731,21 +731,42 @@ def _logger(self):
     def __init__(self, app):
         # Global Desktop Environment
         self._app = app
-        self._independent_variables = {}
         self._independent_design_variables = {}
         self._independent_project_variables = {}
-        self._dependent_variables = {}
         self._dependent_design_variables = {}
         self._dependent_project_variables = {}
 
+    @property
+    def _independent_variables(self):
+        all = {}
+        for k, v in self._independent_project_variables.items():
+            all[k] = v
+        for k, v in self._independent_design_variables.items():
+            all[k] = v
+        return all
```

Comment:
@maxcapodi78 I think you could use the `update()` method here:
```suggestion
        all.update(self._independent_project_variables)
        all.update(self._independent_design_variables)

        return all
```